salesloft/metadata/queryParamStats.json

```diff
@@ -1,13 +1,13 @@
 {
   "meta": {
     "totalObjects": 48,
-    "collectedAt": "2024-06-10"
+    "collectedAt": "2024-08-13"
   },
   "queryParams": [
     {
       "name": "updated_at[gt]",
-      "frequency": 0.3125,
-      "totalObjects": 15,
+      "frequency": 0.3333,
```

Comment:
This stats file could be discontinued. The code is there to recalculate query param usage among all endpoints